helper/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "cudarc_helper"
```

Comment:
This is a nitpick, but thoughts on renaming this binary & the directory to "bindings_generator", or something like that?

Reply:
bindings_generator looks fine to me.